vmdb/app/models/vm_redhat.rb

```diff
@@ -37,4 +37,12 @@ def self.calculate_power_state(raw_power_state)
     else             super
     end
   end
+
+  def supports_migrate?
+    false
+  end
+
+  def supports_reconfigure?
+    false
+  end
```

Comment:
I think the validate_xx methods should be expanded for this use case.  I like your idea of batch validations as well, and it shouldn't be a stretch to create something like batch_validate_xx which returns the ones that pass, and the set of failures in some form.

Not sure we want to do all that in this PR though, but it definitely makes much more sense than doing all that work in the UI.